src/lib/services/workflow-service.ts

```diff
@@ -51,3 +51,67 @@ export async function fetchWorkflow(
     toWorkflowExecution,
   );
 }
+
+export const fetchAllArchivedWorkflows = async (
+  namespace: string,
+  parameters: FilterParameters,
+  request = fetch,
+): Promise<CombinedWorkflowExecutionsResponse> => {
+  const query = toListWorkflowQuery(parameters);
+
+  const { executions, nextPageToken } =
+    (await requestFromAPI<ListWorkflowExecutionsResponse>(
+      routeForApi('archive', { namespace }),
+      {
+        params: { query },
+        request,
+      },
+    )) ?? { executions: [], nextPageToken: '' };
```

Comment:
Should we come up with something better here? Like should `requestFromAPI` take an empty value? What does this request do if Archival isn't enabled?